File: src/main/java/com/janrain/backplane2/server/dao/BusOwnerDAO.java
package com.janrain.backplane2.server.dao;

import com.janrain.backplane2.server.BackplaneServerException;
import com.janrain.backplane2.server.config.Backplane2Config;
import com.janrain.backplane2.server.config.User;
import com.janrain.commons.supersimpledb.SimpleDBException;
import com.janrain.commons.supersimpledb.SuperSimpleDB;
import com.janrain.oauth2.TokenException;
import org.apache.log4j.Logger;

import static com.janrain.backplane2.server.config.Backplane2Config.SimpleDBTables.BP_BUS_OWNERS;

/**
 * @author Johnny Bufu
 */
public interface BusOwnerDAO extends DAO<User> {

}

File: src/main/scala/com/janrain/backplane/server2/dao/BusOwnerDao.scala
package com.janrain.backplane.server2.dao

import com.janrain.backplane.dao.DaoAll
import com.janrain.backplane.common.UserPassAuth
import com.janrain.backplane.server2.oauth2.model.{BusOwnerFields, BusOwner}

/**
 * @author Johnny Bufu
 */
trait BusOwnerDao extends DaoAll[BusOwner] with UserPassAuth[BusOwnerFields.EnumVal, BusOwner]
